test/metabase/driver/postgres/test_data.clj

```diff
@@ -0,0 +1,257 @@
+(ns metabase.driver.postgres.test-data
```

Comment:
New test data loading code currently lives in `metabase.driver.<driver>.test-data` instead of `metabase.test.data.<driver>`. Structuring the pattern for test extension namespaces that way was something I immediately regretted, maybe now we can finally start working towards moving test extensions to something more sane that follows logical package/subpackage groupings